specs/latest/2.0/index.html

```diff
@@ -785,27 +785,29 @@ <h2 class="no-toc">Table of contents</h2>
                             GLsizei height, GLint border, GLintptr offset);
   void compressedTexImage2D(GLenum target, GLint level, GLenum internalformat, GLsizei width,
                             GLsizei height, GLint border, ArrayBufferView srcData,
-                            optional GLuint srcOffset = 0);
+                            optional GLuint srcOffset = 0, optional GLuint srcLengthOverride = 0);
```

Comment:
Also, we need to define what happens if srcOffset + srcLengthOverride > srcData.length. Not sure whether that should be INVALID_OPERATION or INVALID_VALUE. The uniform[1234] entry points generate INVALID_VALUE for this sort of condition. The texImage entry points generate INVALID_OPERATION if srcOffset would mean reading beyond the end of the array.

Also, what happens if srcLengthOverride is longer than srcData.length - srcOffset? Should that be INVALID_OPERATION?